install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -141,6 +141,8 @@ spec:
                         type: string
                     type: object
                 type: object
```

Comment:
@MUzairS15 @KiptoonKipkurui, please add descriptions for the fields in this CRD.